ui/src/containers/Search/useDebounce.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+import { useMst } from '../../store/root';
+
+export const UseDebounce = (value: string, delay: number) => {
+  const [debouncedValue, setDebouncedValue] = useState(value);
```

Comment:
should we move this to a separate directory named utils where all the helper functions can be placed? :thinking: 
/cc @PuneetPunamiya